sos_trades_core/sos_wrapping/test_discs/disc_distortion.py

```diff
@@ -0,0 +1,69 @@
+'''
+Copyright 2022 OS-Climate https://os-climate.org
```

Comment:
what should be a copyright for non-Airbus files like this?

Reply:
None just put Apache license?

Reply:
I'm not used to files without copyright, but allright, we may keep Apache license only